test-install/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+shopt -s extglob
+set -e
+set -o errtrace
+set -o errexit
+set -o pipefail
+
+script_url=https://raw.githubusercontent.com/mysteriumnetwork/node/master/install.sh
```

Comment:
Should this be a local version of `install.sh`? You are pulling always from the master.
```suggestion
script_url=https://raw.githubusercontent.com/mysteriumnetwork/node/master/install.sh
```

Reply:
I chose this so there's no need to mount anything. Users always install it from the master branch anyway.

Reply:
It may change as I am now trying to implement it using containers instead of VM (so we can run it on CI).